qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/shared/OfferOverview.groovy

```diff
@@ -0,0 +1,34 @@
+package life.qbic.portal.qoffer2.shared
+
+/**
+ * <class short description - 1 Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since <versiontag>
+ */
```

Comment:
```suggestion
/**
 * This class holds data for an offer overview
 *
 * It is not always necessary to pass along complete offer data in the application and display this to the user.
 * For this purpose, you can use this class and provide a concise offer overview, 
 * and its content can be loaded quick and easy.
 *
 * @since 1.0.0
 */
```

Reply:
Very nice 👍